.eslintrc.js

```diff
@@ -15,7 +15,7 @@ module.exports = {
     node: true,
     jest: true,
   },
-  ignorePatterns: ['.eslintrc.js', 'jest.config.js'],
+  ignorePatterns: ['.eslintrc.js', 'jest.config.js', 'src/merklize/**/*'],
```

Comment:
No sense run lint.  `merklize` pkg will move to a separate library latter.

Reply:
I would add todo just to not forget to revert this

Reply:
Done